profile/src/main/java/com/scottlogic/deg/profile/reader/file/CSVFromPathToStringsLoader.java

```diff
@@ -0,0 +1,58 @@
+package com.scottlogic.deg.profile.reader.file;
+
+import com.scottlogic.deg.profile.reader.file.inputstream.FilepathToInputStream;
+import com.scottlogic.deg.profile.reader.file.inputstream.PathMapper;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.nio.charset.Charset;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class CSVFromPathToStringsLoader {
```

Comment:
not sure the naming is still appropriate

CsvStreamReader? 

and retrieveNames should probably not refer to names.